views/contact_info.ejs

```diff
@@ -0,0 +1,78 @@
+<% include _header.ejs %>
+
+<div class="container">
+    <div class="col-lg-3"></div>
+
+    <div class="col-lg-6 contact-info-container">
+        <div class="col-lg-12 col-md-12 col-sm-12 col-xs-12">
+            <div class="logo col-lg-6 col-md-6 col-sm-6 col-xs-6">
+                <img src="/public/img/unhangout-logo-blue-full.png">
+            </div>
+        </div>
+
+        
+        <div class="col-lg-12 col-md-12 col-sm-12 col-xs-12">
```

Comment:
This class list is the equivalent of just ``col-xs-12``.  BS3 will use the class that is no larger than the current screen, so just specifying ``xs`` applies to all the others.